src/vs/platform/theme/common/colorRegistry.ts

```diff
@@ -291,6 +291,12 @@ export const quickInputTitleBackground = registerColor('quickInputTitle.backgrou
 export const pickerGroupForeground = registerColor('pickerGroup.foreground', { dark: '#3794FF', light: '#0066BF', hc: Color.white }, nls.localize('pickerGroupForeground', "Quick picker color for grouping labels."));
 export const pickerGroupBorder = registerColor('pickerGroup.border', { dark: '#3F3F46', light: '#CCCEDB', hc: Color.white }, nls.localize('pickerGroupBorder', "Quick picker color for grouping borders."));
 
+/**
+ * Keybinding label
+ */
+export const keybindingLabelBackground = registerColor('keybindingLabel.background', { dark: new Color(new RGBA(128, 128, 128, 0.17)), light: new Color(new RGBA(221, 221, 221, 0.4)), hc: new Color(new RGBA(128, 128, 128, 0.17)) }, nls.localize('keybindingLabelBackground', 'Keybinding label background color. The keybinding label is used in the command palette to represent a keyboard shortcut.'));
+export const keybindingLabelForeground = registerColor('keybindingLabel.foreground', { dark: Color.fromHex('#CCCCCC'), light: Color.fromHex('#555555'), hc: Color.fromHex('#CCCCCC') }, nls.localize('keybindingLabelForeground', 'Keybinding label foreground color. The keybinding label is used in the command palette to represent a keyboard shortcut.'));
```

Comment:
`The keybinding label is used in the command palette to represent a keyboard shortcut.`, not only, also in watermark and in the keybindings editor itself:

![image](https://user-images.githubusercontent.com/900690/113698134-6bc15e80-96d4-11eb-8252-123b867ea8d2.png)